src/material/toolbar/testing/shared.spec.ts

```diff
@@ -0,0 +1,91 @@
+import {Component} from '@angular/core';
+import {ComponentHarness, HarnessLoader} from '@angular/cdk/testing';
+import {MatToolbarModule} from '@angular/material/toolbar';
+import {
+  MatToolbarHarness,
+  MatToolbarSection
+} from '@angular/material/toolbar/testing/toolbar-harness';
+import {ComponentFixture, TestBed} from '@angular/core/testing';
+import {TestbedHarnessEnvironment} from '@angular/cdk/testing/testbed';
+
+export function runHarnessTexts(
```

Comment:
Looks like a typo. Might be also good to have a small comment as we have in other harness tests.

```suggestion
/** <add comment> */
export function runHarnessTests(
```